client/src/components/events-components/EventCard.tsx

```diff
@@ -7,11 +7,12 @@ import { deleteEventRsvp, eventRsvp } from "@/utils/eventService";
 
 interface EventProps {
   event: EventWithRsvp;
-  editable: boolean;
+  eventEditable: boolean;
```

Comment:
nitpick but I think `editable` is fine, given the interface name is `EventProps`